Project.toml

```diff
@@ -11,6 +11,7 @@ DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
+GraphMakie = "1ecd5474-83a3-4783-bb4f-06765db800d2"
```

Comment:
Indeed this was something I wanted to let you know since this is not very good, but before it was worse anyway. I guess another solution is then adding the dependency with something like Pkg.add(...) when AgentsVisualizations is compiled, don't like it very much though

Reply:
but I think with a try catch block we can get around this easily

Reply:
mmmh it's not that simple, need to dig up a bit to solve this. Notice however that this doesn't change the loading time of Agents, only the amount of time needed when it is first installed

Reply:
Hitting a wall at the moment, but is it unacceptable the current way if the only downside is that `add Agents` takes more?

Reply:
Another possibility is to transform AgentsVisualizations in its own package as it is done in https://github.com/MakieOrg/Makie.jl with GLMakie, ecc...

This is a breaking change, but we are going to 6.0 and the addition of these extensions is newish in any case

Reply:
This problem is only about the graph space. Perhaps make another extension that is loadde on GraphMakie? And trhen there is the function that plots the space, that is extended by the plotting extension, and there is a special extension for the graph extension library? in any case, please open a new issue and we discuss this in an open issue?